packages/react-components/react-tabs/src/components/Tab/useTabStyles.ts

```diff
@@ -26,26 +26,31 @@ const useRootStyles = makeStyles({
     ...shorthands.borderRadius(tokens.borderRadiusMedium),
     ...shorthands.borderWidth(0),
     cursor: 'pointer',
-    display: 'flex',
-    flexDirection: 'row',
+    display: 'grid',
     flexShrink: 0,
+    gridAutoFlow: 'column',
+    gridTemplateColumns: 'auto',
+    gridTemplateRows: 'auto',
     fontFamily: tokens.fontFamilyBase,
     lineHeight: tokens.lineHeightBase300,
     outlineStyle: 'none',
     position: 'relative',
     ...shorthands.overflow('hidden'),
     textTransform: 'none',
   },
+  horizontal: {
+    justifyContent: 'center',
+  },
+  vertical: {
+    justifyContent: 'flex-start',
```

Comment:
this is bit confusing, can we use grid apis? 
```suggestion
    justifyContent: 'start',
```